File: CP003/Java/Main.java
package fosalgo;
import java.util.Scanner;
public class Main {
    
    public static void main(String[] args) {
        int a, b;
        Scanner sc = new Scanner(System.in);
        a = sc.nextInt();
        b = sc.nextInt();
        int lower = a + 1;
        int upper = b - 1;
        if (b < a) {
            lower = b + 1;
            upper = a - 1;
        }
        for (int i = lower; i <= upper; i++) {
            System.out.print(i+" ");
        }
    }
}

File: CP003/C++/Main.cpp
#include <iostream>

using namespace std;
int main() {
    int a,b;
    cin >> a;
    cin >> b;
    int lower = a + 1;
    int upper = b - 1;
    if(b<a){
        lower = b + 1;
        upper = a - 1;
    }
    for(int i = lower; i<=upper;i++){
        cout << i <<" ";
    }
    return 0;
}
